dlt/common/runtime/typing.py

```diff
@@ -0,0 +1,18 @@
+from typing import Any, Callable, Dict, List, Literal, Optional, Sequence, Set, Type, TypedDict, NewType, Union, get_args
+
+
+TExecInfoNames = Literal["kubernetes", "docker", "codespaces", "github_actions", "airflow", "notebook", "colab","aws_lambda","gcp_cloud_function"]
```

Comment:
could you format it? or maybe we should go back to blake formatter initially in the same mode we have in verified sources?

Reply:
i am also for black formatter..